_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -545,15 +573,23 @@ GET _plugins/_transform?size=2&from=8
 Introduced 1.0
 {: .label .label-purple }
 
-Transform jobs created using the API are automatically enabled, but if you ever need to enable a job, you can use the `start` API operation.
+Transform jobs created using the API are automatically enabled, but if you ever need to enable a job, you can use the start API operation. 
```

Comment:
Do we mean "started" and "start" instead of "enabled" and "enable"?

Reply:
This is tricky because technically they are run on a schedule, so they are not "started" per se. I think leaving it "enabled" makes it clearer that it is enabled, but it will run when it's scheduled to run, and not right away.